controller_manager/src/controller_manager.cpp

```diff
@@ -62,33 +63,52 @@ ControllerManager::ControllerManager(
   loader_(std::make_shared<pluginlib::ClassLoader<controller_interface::ControllerInterface>>(
       kControllerInterfaceName, kControllerInterface))
 {
+  deterministic_callback_group_ = create_callback_group(
+    rclcpp::callback_group::CallbackGroupType::MutuallyExclusive);
```

Comment:
this callback group is not being used within the controller manager, am I seeing this right?
If so, I'd remove this and have the realtime threads being handled outside of the CM.

Reply:
This comment is superseded by your other related comment, https://github.com/ros-controls/ros2_control/pull/216/files#r518216619. I'll resolve this one to reduce discussion noise.